Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem "jazzy"
```

Comment:
If we're adding a Ruby environment, can you add `cocoapods` to the `Gemfile`, and add `.ruby-version` (2.4.1) and `.ruby-gemset` (alamofire) files too? That way it will integrate with RVM / rbenv automatically.